dirwatcher.py

```diff
@@ -1,56 +1,92 @@
 __author__="Sasha Lukas + demo"
 
-import logging 
-import datetime
-import time
-import argparse
 import os
+import argparse
+import time
+import datetime
+import logging
+import signal
+
 logger = logging.getLogger(__file__)
+exit_flag = False
+files_found = []
+magic_word_position = {}
```

Comment:
Variable naming:  this dictionary tracks the last line that was read in the file, it does not actually track position of magic_words ... perhaps `file_line_positions` ?